build.cmd.ps1

```
@@ -16,14 +16,14 @@ if (-not ($?)) {exit(1)}
 $internalversion = & $pythonexe ..\salt\salt\version.py msi 
 if (-not ($?)) {exit(1)}
 
-if (-not ($displayversion -match '^[\d\.]{6,18}$')) {
-  Write-Host -ForegroundColor Red $displayversion is not a version
-  exit(1)
-}
-if (-not ($internalversion -match '^\d\d\.[\d]{1,2}\.[\d]{1,2}$')) {
-  Write-Host -ForegroundColor Red $internalversion is not a valid msi version
-  exit(1)
-}
+#if (-not ($displayversion -match '^[\d\.]{6,18}$')) {
+#  Write-Host -ForegroundColor Red $displayversion is not a version
+#  exit(1)
+#}
+#if (-not ($internalversion -match '^\d\d\.[\d]{1,2}\.[\d]{1,2}$')) {
+#  Write-Host -ForegroundColor Red $internalversion is not a valid msi version
+#  exit(1)
+#}
```

Need to figure out what we want to do here

---

Both regexp checks are a question of taste.
They serve to "fail fast" on bad input.

The msi compiler will stop and complain an illegally formed `$internalversion` with a clear and precise error message, so remove the check here, if you want to save lines of code.

---

Could you please paste the error messages when you use the 3000 versioning?

---

Same here
With https://github.com/markuskramerIgitt/salt-windows-msi/commit/0aec90c5884adcdea6bd5fac90fd7569a3684c65 I tested the msi build on 

    Salt-Minion-3000-Py3-AMD64-Setup.exe
    Salt-Minion-3000-Py2-AMD64-Setup.exe
    Salt-Minion-3000-Py3-x86-Setup.exe

Please revert your commit, not need for a change.